### DISCUSSION TITLE
hello guys, my mine crashes when i put iron-jetpacks could you tell me the reason?

### DISCUSSION THREAD

**Discussion by gutaozzeira** [shedaniel/RoughlyEnoughItems#1261] [Q & A] [answered]
hello guys, my mine crashes when i put iron-jetpacks could you tell me the reason?

[crash-2022-12-07_14.09.20-client.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/10178871/crash-2022-12-07_14.09.20-client.txt)

**Comment by shedaniel** [comment, marked as answer]
This is the discussion page for REI
